lib/stash.rb

```diff
@@ -0,0 +1,170 @@
+=begin
+stash.rb: Core lich file for extending free_hands, empty_hands functions in
+  item / container script indifferent method.  Usage will ensure no regex is
+  required to be maintained.
+
+    Maintainer: Elanthia-Online
+    Original Author: Tillmen, Ondreian, others
+    game: Gemstone
+    tags: CORE, spells
+    required: Lich > 5.0.19
+    version: 1.0.0
+
+  changelog:
+    version 1.0.0
+     * Initial release
+
+=end
+
+module Lich
+  module Stash
+    def self.find_container(param)
+      GameObj.inv.find do |container|
+        container.name =~ %r[#{param}]
+      end or fail "could not find Container[name: #{param}]"
+    end
+
+    def self.container(param)
+      container = find_container(param)
+      if container.contents.nil?
+        # opening a closed container populates the container's inventory in XML.
```

Comment:
This can also be done with a simple look, which is 1 command less (in most cases)
We would need a fallback for when it's closed